Configuration/src/ConfigurationProviders/Tiltfile

```diff
@@ -2,7 +2,7 @@ LOCAL_PATH = os.getenv("LOCAL_PATH", default=".")
 NAMESPACE = os.getenv("NAMESPACE", default="apps")
 WAIT_TIMEOUT = os.getenv("WAIT_TIMEOUT", default='10m00s')
 TYPE = os.getenv("TYPE", default='web')
-NAME = "steeltoe-samples-configuration"
+NAME = "configuration-providers-sample"
```

Comment:
I don't know whether the changes made to this file are correct.

I wonder if we should keep this file. Is it hard to produce, or auto-generated?
It feels inconsistent that only one sample has it.

Reply:
The values here just need to match the name in workload.yaml, the changes should be fine. It isn't extremely hard to produce (there's a template for it in the TAP Visual Studio extension) but is not currently autogenerated. There were plans to make it go away entirely, but I don't think that's going to happen anymore.

It is inconsistent for only one to have it... any sample that is expected to work on some TAP should have one, and I'd rather see us head towards more having it than not.

Reply:
Isn't the TAP extension currently broken, without having a path forward on how it can be fixed?

Reply:
Talked about this offline, removing it for now.